app/layout.tsx

```diff
@@ -7,6 +7,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
+
     <SessionWrapper>
```

Comment:
SessionWrapper add in to inside body element

Reply:
i have put it inside and commited it into the pull request